src/main/java/frc5124/robot2020/subsystems/Turret.java

```diff
@@ -7,11 +7,24 @@
 
 package frc5124.robot2020.subsystems;
 
+import com.revrobotics.CANPIDController;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
 import edu.wpi.first.wpilibj2.command.Subsystem;
+import frc5124.robot2020.RobotMap;
 
 public class Turret implements Subsystem {
+
+  private final NetworkTable limelight;
+  private final CANSparkMax motor;
   
   public Turret() {
+    limelight = NetworkTableInstance.getDefault().getTable("limelight");
```

Comment:
the command or command group should get the target position from network table and pass to the subsystem. The subsystem should only be responsible for running to and holding the target position